tests/components/mqtt/conftest.py

```diff
@@ -1,3 +1,7 @@
 """Test fixtures for mqtt component."""
+import logging
+
 from tests.components.blueprint.conftest import stub_blueprint_populate  # noqa: F401
 from tests.components.light.conftest import mock_light_profiles  # noqa: F401
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Is the mqtt integration tests asserting debug level logs?

Reply:
It is

Reply:
May be some other integrations are still doing that. Those tests need to be updated first.

Reply:
This is ok as a stop gap. Preferably we should avoid asserting logs at debug level. Asserting logs in integrations is generally good to avoid as it's not robust.